### DISCUSSION TITLE
Auto-lock old closed issues?

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#14114] [Governance]
There are times (e.g. most recently [here](https://github.com/bokeh/bokeh/pull/7604#issuecomment-2428005830)) where I don't seem to be able to successfully convey that discussion on an old issue should cease immediately, and and new issue should be opened to continue future discussion. If the (years) old issue had simply been auto-locked to in the first place, then presumably the user would have created a new issue to begin with.

I'd like to propose auto-locking all closed issues after some period of inactivity (three months, six months, whatever). I am certain there are some GH integrations to automate this, and I also believe from experience that this sort of auto-locking policy is fairly common. Thoughts @bokeh/dev?

**Comment by mattpap** [comment]
I'm all for locking closed issues after 3 months. That should be plenty of time to raise any immediate concerns and perhaps reopen an issue if needed.

**Comment by ianthomas23** [comment]
:+1: from me.

**Comment by Azaya89** [comment]
Aye!

**Comment by bryevdv** [comment]
Seems like there's pretty broad agreement. I'll bring it up in the meeting tomorrow but go ahead and start looking into options. A couple that I found immediately:

* https://github.com/marketplace/actions/lock-threads
* https://github.com/marketplace/actions/inactivity-lock

If anyone has any other specific suggestions for integrations to use, please link them in replies.

**Reply by mattpap** [reply]
Regarding failing `Lock threads` workflow (with "Resource not accessible by integration") see https://github.com/dessant/lock-threads/issues/47#issuecomment-2041019276. I started PR #14127 to address this.
